lib/resty/memcached.lua

```diff
@@ -736,4 +807,51 @@ function _M.close(self)
 end
 
 
+function _M.init_pipeline(self, n)
+    if self._reqs then 
+        return "already init pipeline"
+    end
+
+    if n and type(n) ~= 'number' then
+        return "buffer size is number type"
+    end
+    self._reqs = {
+        table.new(n or 4, 0),
```

Comment:
we can define `table.new` as a global value in this lib, like here: https://github.com/openresty/lua-resty-upstream-healthcheck/blob/master/lib/resty/upstream/healthcheck.lua#L36

Reply:
changed